docs/user/aws/install.md

```diff
@@ -48,6 +49,13 @@ In EC2, there will be 6 running instances:
 
 ![EC2 instances after install](images/install_nodes.png)
 
+The installation creates an encrypted AMI for the bootstrap and control-plane machines.
+The encrypted AMI is [copied][encrypted-copy] from the AMI configured in the control-plane machine-API provider spec,
+which is RHCOS by default. 
```

Comment:
nit: more whitespace issues on this line and others:

```git
$ git show --check --oneline origin/pr/1408
b29c9736a (origin/pr/1408) Adding initial user doc/guide & materials for UPI AWS installation
docs/user/aws/install.md:54: trailing whitespace.
+which is RHCOS by default. 
docs/user/aws/install.md:55: trailing whitespace.
+The encryption uses the default EBS key for your target account and region 
docs/user/aws/install.md:58: trailing whitespace.
+ 
docs/user/aws/install_upi.md:87: trailing whitespace.
+INFO Waiting up to 30m0s for the Kubernetes API... 
docs/user/aws/install_upi.md:88: trailing whitespace.
+INFO API v1.12.4+c53f462 up                       
docs/user/aws/install_upi.md:106: trailing whitespace.
+$ export KUBECONFIG=auth/kubeconfig 
docs/user/aws/install_upi.md:131: trailing whitespace.
+By default, MachineSets are created and will have failed to launch. We can correct the desired subnet filter, 
docs/user/aws/install_upi.md:228: trailing whitespace.
+Once the Machine API has a chance to reconcile and begin launching hosts with the correct attributes, you 
docs/user/aws/install_upi.md:242: trailing whitespace.
+scaling groups. 
```

Reply:
Fixed.